src/components/HomePage/LinksSection.jsx

```diff
@@ -31,32 +31,27 @@ const linkItems = [
     },
     {
         text: "CONTACT",
-        path: ""
+        path: "mailto:DIALUPSTUFF@GMAIL.COM"
     }
 ];
 
 class LinkItem extends React.Component {
+
     openContactAlert() {
         let contactString = "EMAIL US AT DIALUPSTUFF@GMAIL.COM\n\nFACEBOOK AT HTTPS://WWW.FACEBOOK.COM/DIALUPSTUFF";
         alert(contactString);
     }
+
     render() {
-        let target = this.props.text.toLowerCase().replace(/\s/g,'');
-        return this.props.text == "CONTACT"
-            ?(
-                <div className={cx('linkItem')}>
-                    <a style={{"color": this.props.color}} onClick={this.openContactAlert}>
-                        <p>{this.props.text}</p>
-                    </a>
-                </div>
-            )
-            :(
-                <div className={cx('linkItem')}>
-                    <a style={{"color": this.props.color}} target={target} href={this.props.path}>
-                        <p>{this.props.text}</p>
-                    </a>
-                </div>
-            )
+        const contactLink = this.props.text === "CONTACT";
```

Comment:
It seems like we don't really need this any more, since we don't need to differentiate between the contact link and other links in this section. I think we can remove this line, and then set the target directly.

Reply:
So I was running into a case where:

leaving all the targets as being `this.props.text.toLowerCase().replace(/\s/g,'')` will result in there being an extra, empty window that opens when you click on the contact link (since the contact link doesn't have the its `href` set to an actual URL). Conditionally setting the target as null, though, will still open the mailto link, just without that extra window

Will definitely continue to investigate to see if this is really necessary, though!

Reply:
Ahh okay makes sense. In that case, can we rename this to `isContactLink` or place this directly into the ternary?